hw/arm/aspeed.c

```diff
@@ -369,6 +370,24 @@ static void aspeed_machine_init(MachineState *machine)
                          amc->uart_default);
     qdev_realize(DEVICE(&bmc->soc), NULL, &error_abort);
 
+    SpiGpioState *spi_gpio = SPI_GPIO(qdev_new(TYPE_SPI_GPIO));
+    qdev_realize(DEVICE(spi_gpio), NULL, &error_fatal);
+
+    DeviceState *m25p80 = qdev_new("n25q256a");
+    qdev_realize(m25p80, BUS(spi_gpio->spi), &error_fatal);
+
+    spi_gpio->cs_line = qdev_get_gpio_in_named(m25p80, SSI_GPIO_CS, 0);
+    qdev_connect_gpio_out_named(DEVICE(&bmc->soc.gpio),
+                                "sysbus-irq", 185, spi_gpio->cs_line);
```

Comment:
```suggestion
    qdev_connect_gpio_out_named(DEVICE(&bmc->soc.gpio),
                                "sysbus-irq", 185, 
                                qdev_get_gpio_in_named(m25p80, SSI_GPIO_CS_in, 0));
```
Also considering MACRO, which converting the GPIO name to this number.  i.e. AST_GPIO_2_IRQ_NUM('X', 0) 
This way the code will tell we connect GPIOX0 as SPI_GPIO_CS input for our device.
all the SPI_GPIO input pin connecting to bmc->soc.gpio code can follow this pattern.